pydicer/convert/convert_pt.py

```diff
@@ -0,0 +1,311 @@
+        sitk_writer (sitk.WriteImage(), optional): The SimpleITK object used
+                                                   to write an array to the
+                                                   NIFTI format. Defaults to
+                                                   None.
+        patient_weight_from_ct (float, optional): If the patient's weight is
+                                                  missing from the PT DICOM
+                                                  it can be provided through
+                                                  this argument. Defaults to
+                                                  None.
+        dtype_image (numpy.dtype, optional): The dtype in which to save the
+                                             image. Defaults to np.float32.
+    Raises:
+        MissingWeightException: Error to alert when the weight is missing from
+                                the PT, to compute the SUV.
+        RuntimeError: Error to alert when one or more slices are missing
+        ValueError: Raised when a modality or a unit (for the PT) is not
+                    handled.
+    Returns:
+        numpy.array: The numpy image, used to compute the bounding boxes
+    """
+    slices = [pdcm.read_file(str(dcm["path"])) for dcm in input_filepaths]
+    slices.sort(key=lambda x: float(x.ImagePositionPatient[2]))
+    if modality == "PT":
+        if hasattr(slices[0], "PatientWeight") and (slices[0].PatientWeight is None):
+            if hasattr(slices[0], "PatientsWeight"):
+                patient_weight = float(slices[0].PatientsWeight)
+            elif patient_weight_from_ct is not None:
+                patient_weight = patient_weight_from_ct
+            else:
+                # raise MissingWeightException(
+                #     'Cannot compute SUV the weight is missing')  # from Shuchao's code
+                patient_weight = 75.0  # From Shuchao's code (default)
+                warnings.warn(
+                    "Cannot find the weight of the patient, hence it "
+                    "is approximated to be 75.0 kg"
+                )
+        elif not hasattr(slices[0], "PatientWeight"):
+            if hasattr(slices[0], "PatientsWeight"):
+                patient_weight = float(slices[0].PatientsWeight)
+            elif patient_weight_from_ct is not None:
+                patient_weight = patient_weight_from_ct
+            else:
+                # raise MissingWeightException(
+                #     'Cannot compute SUV the weight is missing')  # from Shuchao's code
+                patient_weight = 75.0  # From Shuchao's code (default)
+                warnings.warn(
+                    "Cannot find the weight of the patient, hence it "
+                    "is approximated to be 75.0 kg"
+                )
+        else:
+            patient_weight = float(slices[0].PatientWeight)
+
+    # Check if all the slices come from the same serie
+    same_serie_uid = True

```

Comment:
I know it's tempting to drop `s` to make it singular but that doesn't really work because `serie` isn't a word. In my opinion, we should use the full word `series` throughout to ensure correct spelling of all our variables.